sdk/core/Azure.Core/src/FixedDelayStrategy.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Azure.Core
+{
+    /// <summary>
+    ///
+    /// </summary>
+    internal class FixedDelayStrategy : DelayStrategy
+    {
+        private readonly TimeSpan _delay;
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="delay"></param>
+        public FixedDelayStrategy(
+            TimeSpan delay) : base(TimeSpan.FromMilliseconds(delay.TotalMilliseconds * 1.2), 0.8, 1.2)
```

Comment:
Should we be explicit about `MaxDelay` here rather than implicitly applying the jitter?

Reply:
Yeah we probably could do that, but the issue I was trying to avoid was having the jitter be ignored half the time.

Reply:
Yeah, that is what made me think about how this should be specified. It seems that we should either be explicit or document that the `maxDelay` is always subject to jitter adjustments and expose that with a default.

Reply:
I changed to keep it maxDelay as absolute.